src/alr/alr-commands-update.adb

```diff
@@ -113,9 +147,23 @@ package body Alr.Commands.Update is
    function Long_Description (Cmd : Command)
                               return Alire.Utils.String_Vector is
      (Alire.Utils.Empty_Vector
-      .Append ("Resolves dependencies using the loaded indexes, and"
-               & " regenerates the aggregate project building file found in"
-               & " <crate>" & GNAT.OS_Lib.Directory_Separator
-               & Alire.Paths.Working_Folder_Inside_Root));
+      .Append ("Resolves unpinned dependencies using available indexes"));
```

Comment:
Is the "selective" per crate update explain in the help message?

Reply:
It is now